lib/graphql/persisted_queries/store_adapters/redis_store_adapter.rb

```diff
@@ -7,22 +7,27 @@ module PersistedQueries
     module StoreAdapters
       # Redis adapter for storing persisted queries
       class RedisStoreAdapter < BaseStoreAdapter
-        def initialize(redis_client:)
+        DEFAULT_EXPIRATION = 24 * 60 * 60
+        DEFAULT_NAMESPACE = "graphql-persisted-query"
+
+        def initialize(redis_client:, expiration: DEFAULT_EXPIRATION, namespace: DEFAULT_NAMESPACE)
           @redis_proc = build_redis_proc(redis_client)
+          @expiration = expiration
+          @namespace = namespace
         end
 
         def fetch_query(hash)
           @redis_proc.call { |redis| redis.get(key_for(hash)) }
         end
 
         def save_query(hash, query)
-          @redis_proc.call { |redis| redis.set(key_for(hash), query, ex: 24 * 60 * 60) }
+          @redis_proc.call { |redis| redis.set(key_for(hash), query, ex: @expiration) }
         end
 
         private
 
         def key_for(hash)
-          "graphql-persisted-query:#{hash}"
+          [@namespace, hash].join(":")
```

Comment:
If I'm not mistaken, "#{@namespace}:#{hash}" will be a bit more effective, because it doesn't initiate the additional array 🙂

Reply:
Yeah, had a similar discussion with a co-worker.  Went with this way thinking that if I implement the feature to cache the `Query` objects instead, I'd have more things in there (like the GraphQL gem version), but that's probably getting ahead of myself.  I'll simplify it for now.